dev/env/scripts/lib.sh

```diff
@@ -315,7 +316,21 @@ docker_logged_in() {
 }
 
 load_external_config() {
-  local chamber="${CHAMBER:-$CHAMBER_DEFAULT}"
+  local chamber
+  local use_aws_vault="${USE_AWS_VAULT:-$USE_AWS_VAULT_DEFAULT}"
+  if [ "$use_aws_vault" = true ]; then
+    local profile="${AWS_VAULT_PROFILE:-$AWS_VAULT_PROFILE_DEFAULT}"
+    local credentials_exist
+    credentials_exist=$(aws-vault list --credentials | grep "${profile}" || true)
+    if [[ ! $credentials_exist ]]; then
+      log "Creating AWS Vault profile '$profile'"
+      aws-vault add "$profile"
+    fi
+    chamber="aws-vault exec ${profile} -- chamber"
```

Comment:
nit: I guess, if the user would accidentally store a value containing a whitespace in `AWS_VAULT_PROFILE`, then this would probably fail with some usage error.
```suggestion
    chamber="aws-vault exec '${profile}' -- chamber"
```

Reply:
I've just checked, with single quotes it doesn't work, but works with a whitespace